tests/regression/56-witness/05-prec-problem.c

```diff
@@ -0,0 +1,22 @@
+//PARAM: --enable witness.yaml.enabled --enable ana.int.interval
+#include <stdlib.h>
+
+int foo(int* ptr1, int* ptr2){
+    int result;
+    if(ptr1 == ptr2){
+        result = 0;
+    } else {
+        result = 1;
+    }
+    // Look at the generated witness.yml to check whether there are contradictory precondition_loop_invariant[s]
```

Comment:
This isn't great as a test, but without something like #787 we cannot do any better either.